source/content/guides/composer-convert.md

```diff
@@ -190,49 +247,49 @@ composer prepare-for-pantheon
+This modifies the `.gitignore` file and cleans up any errant `.git` directories in the codebase to prepare your new code for direct deployment to Pantheon.
 
-## Commit
+## Deploy
 
-Commit your work to the git repo. From the `$site` directory, run the following:
+You've now committed the code to the local branch. If your site has Multidev, you can deploy that branch directly to a new Multidev and test the site in the browser. If the site doesn't load properly, clear the cache. If there are any issues, utilize your site's logs via `terminus drush $site.composerify -- wd-show` to inspect the watchdog logs, or follow the directions in our documentation on [log collection](/logs).
 
-```bash{promptUser: user}
-cp -r .git ../$site-composer/
-cd ../$site-composer
+### Deploy to a Multidev
+
+If your site has Multidev, push the changes to a Multidev called `composerify` to safely test the site without affecting the Dev environment:
+
+```bash{promptUser:user}
 git add .
-git commit -m "Convert to Composer based install"
+git commit -m "ran composer prepare-for-pantheon and install"
+git push origin composerify
 ```
 
-You should see a large amount of files committed to the new branch we created earlier.
+Once you have confirmed the site is working, merge `composerify` into `master`, and follow the standard workflow to QA a code change before going live.
 
-## Deploy
+### Deploy to Dev
 
-You've now committed the code to a branch. If your site has Multidev, you can deploy that branch directly to a new Multidev and test the site in the browser. If the site doesn't load properly, clear the cache. If there are any issues, utilize your site's logs via `terminus drush $site.composify -- wd-show` to inspect the watchdog logs, or follow the directions on our documentation on [log collection](/logs).
+If you have a [local development](/local-development) solution, consider testing your `composerify` branch locally before merging.
 
-Once you have confirmed the site is working, merge `composify` into `master`, and follow the standard workflow to QA a code change before going live.
+To force push the changes from the local branch to the `master` Dev branch:
 
-If your plan does not include Multidev, you will have to merge to master before deploying, then follow the rest of the steps above. If you have a local development solution, consider testing your `composify` branch locally before merging.
+```bash{promptUser:user}
+git add .
+git commit -m "ran composer prepare-for-pantheon and install"
+git push -f origin composerify:master
+```
 
 ## Change Upstreams
 
-Your Pantheon site is no longer compatible with traditional upstream updates. Avoid confusion by moving your site to an empty upstream:
+Set the site to use the Drupal 9 Upstream:
 
 ```bash{promptUser:user}
-terminus site:upstream:set $site empty
+terminus site:upstream:set $site drupal9
```

Comment:
It should be safe to do this even if the user pinned to Drupal 8.

Reply:
should I just add the line `It should be safe to do this even if the site is pinned to Drupal 8.`? or is there more nuance here that I'm missing?